Sources/Models/BlogPosts/BlogPost0020_PodcastRSS.swift

```diff
@@ -21,18 +21,31 @@ favorite podcast app!
 
 ### Public Episodes Feed
 
-We have one feed that is open to the public, and you can find it [here](\(path(to: .feed(.episodes)))).
+We have one feed that is open to the public, and you can find it [here](/feed/episodes.xml).
 Plug that into any podcast app and you should have up-to-date access to all of our videos. You will
 get access to full videos of all our free episodes (currently 9 of em) and trailers for the subscriber-only
 episodes. If you find any of our content enticing, then you may want to consider a
-[subscription](\(path(to: .pricing(nil, expand: nil)))) 😃.
+[subscription](/pricing) 😃.
 
 ### Private Episode Feed
 
 If you are a subscriber, then we have one more additional RSS feed for you, and you'll find it on your
-[account](\(path(to: .account(.index)))) page. This gives you full access to _every_ episode on Point-Free
+[account](/account) page. This gives you full access to _every_ episode on Point-Free
 as they are released. We ask that you please do not share this URL with anyone, as it is private and tied
 directly to your account.
+""",
+      timestamp: nil,
+      type: .paragraph
+    ),
+    .init(
+      content: """
+Since the time we launched this feature we have changed this slightly. Only yearly subscribers get access to the entire catalogue of episodes in their podcast RSS feed, and monthly subscribers get access to the last 4 episodes.
+""",
+      timestamp: nil,
+      type: .correction
```

Comment:
went ahead and added this correction to the blog post while I was here:

![image](https://user-images.githubusercontent.com/135203/53176917-0f1a8c80-35bd-11e9-81cb-db2385212501.png)